fast64_internal/sm64/settings/constants.py

```diff
@@ -31,4 +31,5 @@
     ("Refresh 16", "Refresh 16", "Refresh 16"),
     ("", "HackerSM64", ""),
     ("HackerSM64 2.3.0", "HackerSM64 (v2.3.0)", "HackerSM64 (v2.3.0)"),
+    ("HackerSM64 3.0", "HackerSM64 (v3.0)", "HackerSM64 (v3.0)"),
```

Comment:
I would call this 3.0.0, if not just so checking patch version is more trivial/consistent if necessary